dashboard/test/models/regional_partner_test.rb

```diff
@@ -19,7 +19,8 @@ class RegionalPartnerTest < ActiveSupport::TestCase
         city: 'Seattle',
         state: 'WA',
         zip_code: '98101',
-        phone_number: '555-111-2222'
+        phone_number: '555-111-2222',
+        is_active: true
```

Comment:
I think generally, this would also be part of a follow-up PR so that this PR contains only the migration and what "has" to change from running the migration –– the schema and annotations. But I don't think it's worth a whole new Drone build for this, as it's so minor. (I'm went back to this recent comment https://github.com/code-dot-org/code-dot-org/pull/50603#discussion_r1128470594 for reference)

Reply:
Got it, thanks for the note Meg, I'll keep this in mind for future PRs.